_search-plugins/sql/index.md

```diff
@@ -1,76 +1,17 @@
 ---
 layout: default
-title: SQL
+title: SQL Plugin - SQL & PPL
```

Comment:
i don't see other plugin titles having `Plugin` in them, should this just say SQL & PPL?